tests/lib/rule-tester/rule-tester.js

```diff
@@ -2232,6 +2232,268 @@ describe("RuleTester", () => {
         });
     });
 
+    describe("fatal test cases", () => {
```

Comment:
Moved to draft since it looks like I need to add more test cases to ensure all these scenarios work correctly:

* empty schema, validation error when options are passed
* present schema that has one validation error
* present schema that has multiple validation errors